contrib/userscript/unnoticed.user.js

```diff
@@ -0,0 +1,201 @@
+// ==UserScript==
+// @name         Unnoticed Leaderboards
+// @namespace    https://github.com/christopher-dG/unnoticed
+// @version      0.1
+// @description  Display unranked leaderboard entries gathered by Unnoticed on their respective beatmap pages
+// @author       Node
+// @match        https://osu.ppy.sh/*/*
+// @include      https://p9bztcmks6.execute-api.us-east-1.amazonaws.com/*
+// @grant        GM_xmlhttpRequest
+// @require      https://code.jquery.com/jquery-3.2.1.min.js
+// @require      http://momentjs.com/downloads/moment.min.js
+// @connect      p9bztcmks6.execute-api.us-east-1.amazonaws.com
+// @run-at document-end
+// ==/UserScript==
+
+(function(){
+    'use strict';
+
+    var mods_enum = {
```

Comment:
Can we add the `_Key` mania mods as `(N)K` and `TK` for 10K?

Reply:
I will add those once supporting other modes.